dashboard/app/models/pd/teacher_application.rb

```diff
@@ -95,9 +95,15 @@ def validate_required_application_fields
     end
   end
 
+  before_validation :downcase_emails
+  def downcase_emails
```

Comment:
I don't think so. We treat them as separate emails everywhere else in the system, and we often leverage this for multiple test accounts, e.g. `andrew+teacher@code.org` vs. `andrew+student@code.org`